source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -213,10 +213,11 @@ lldb::REPLSP SwiftREPL::CreateInstanceFromDebugger(Status &err,
   debugger.StartEventHandlerThread();
 
   // Destroy the process and the event handler thread after a fatal error.
-  CleanUp cleanup{[&] {
-    process_sp->Destroy(/*force_kill=*/false);
-    debugger.StopEventHandlerThread();
-  }};
+  auto cleanup =
+    llvm::make_scope_exit([&]() {
+        process_sp->Destroy(/*force_kill=*/false);
+        debugger.StopEventHandlerThread();
+        });
```

Comment:
Is this clang-formatted?

Reply:
It is now. (I didn't have clang-format.py on the devbox I pushed this from :-(